targets/pna_nic/pnic_CLI.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
```

Comment:
This file needs to be cleaned up as there are references to multicast / PRE

Reply:
```
def __init__(self, pre_type, standard_client, mc_client, pnic_client):
    runtime_CLI.RuntimeAPI.__init__(self, pre_type,
                                    standard_client, mc_client)
```

The `pre_type` and `mc_client` are required to initialize the RuntimeAPI class. Currently, we can set `pre_type` to `none`. Later, we need to decide the value for it.

But `mc_client` is obtained from the output of the `utils.thrift_connect` function:
```
standard_client, mc_client, pnic_client = runtime_CLI.thrift_connect(
    args.thrift_ip, args.thrift_port, services
)
```
https://github.com/p4lang/behavioral-model/blob/89691ff781b9f5b1f39068e6cccfc2dda18db3d1/tools/bmpy_utils.py#L84

Reply:
@antoninbas, should we set the `pre_type` value to `none` and leave `mc_client` unchanged for now?

Reply:
Yes, I think you need to change `args.pre = runtime_CLI.PreType.SimplePreLAG` in `main` so that it uses the `None` type. That should be enough.